src/seq_mv/csr_spgemm_device_rowbound.c

```diff
@@ -151,7 +151,7 @@ void csr_spmm_symbolic(HYPRE_Int  M, /* HYPRE_Int K, HYPRE_Int N, */
 
    char failed = 0;
 
-#if DEBUG_MODE
+#ifdef HYPRE_DEBUG
    assert(blockDim.z              == NUM_WARPS_PER_BLOCK);
```

Comment:
Should these all be hypre_assert?

Reply:
Yes, they should be but `hypre_assert` is just a print and I want the program to abort at some cases